WangscapeTest/TestPowIntSigned.cpp

```diff
@@ -0,0 +1,102 @@
+#include <gtest/gtest.h>
+
+#include <type_traits>
+
+#include <utils.h>
+
+template<typename I>
+class TestPowIntSigned : public ::testing::Test
+{
+protected:
+    const static I min = std::numeric_limits<I>::min();
+    const static I minus_five = -5;
+    const static I minus_three = -3;
+    const static I minus_two = -2;
+    const static I minus_one = -1;
+    const static I zero = 0;
+    const static I one = 1;
+    const static I two = 2;
+    const static I three = 3;
+    const static I five = 5;
+    const static I digits = std::numeric_limits<I>::digits;
+    const static I max = std::numeric_limits<I>::max();
```

Comment:
Oh, `const`, `static`... Is `static` needed? I would resign from these names. Checking limits, one negative number, one positive, `0`, `1`, `-1` should be enough.